src/tools/melt/MeltStrings.resx

```diff
@@ -134,12 +134,16 @@
    -wx[N]     treat all warnings or a specific message ID as an error
               (example: -wx1059 -wx1067)
    -wxall     treat all warnings as errors (deprecated)
-   -x <path>  export binaries from database to this path
+   -x &lt;path&gt;  export binaries from database to this path
               (defaults to output files path)
+   -xn        export contents in much the same manner as dark.exe
```

Comment:
Don't want version references if we can avoid. Recommend "export contents of File, Binary, and Icon tables to subdirectories to support patching all types of files."